src/edu/mills/cs180a/RomanNumeralTest.java

```diff
@@ -1,218 +1,248 @@
-	void testConvertFromStringToIntThrowsExceptionForInvalidEntry(String s) {
-		assertThrows(IllegalArgumentException.class,
-				() -> convertFromStringToInt(s));
-	}
-
-	@ParameterizedTest
-	@CsvSource({"L,50","XCIV,94","D,500","X,10", "IV,4", "CD,400", "v,5", "xl,40", "IM,1001", "CCCXCVIII,398", "MMMMMMMMM,9000","MMMMMMMMMCMXCIX,9999"})
-	void convertFromStringToIntShouldGenerateTheExpectedIntValue(String input, int expected) {
-		int actualValue = convertFromStringToInt(input);
-		assertEquals(expected, actualValue);
-	}
-
-	@ParameterizedTest
-	@CsvSource({"50,L","94,XCIV","100,C", "9,IX", "40,XL", "1,I", "324,CCCXXIV", "9000,MMMMMMMMM", "9999,MMMMMMMMMCMXCIX","500,D"})
-	void convertFromIntToStringShouldGenerateTheExpectedStringValue(int input, String expected) {
-		String actualValue = convertFromIntToString(input);
-		assertEquals(expected, actualValue);
-	}
-
-	//String to int tests
-	@ParameterizedTest
-	@ValueSource(ints = {-100, 0, 5, 10000})
-	void testIntInputIsOutOfBounds(int invalidInt) {
-		assertThrows(IllegalArgumentException.class,
-				() -> new RomanNumeral(invalidInt));
-	}
-
-	// Testing for convertFromStringToInt
-	@ParameterizedTest
-	@CsvSource({"I,1", "V,5", "X,10", "L,50", "C,100", "D,500", "M,1000"})
-	void testStringToIntValidRomanSymbol(String romanNumeral, int expected) {
-		int actual = convertFromStringToInt(romanNumeral);
-		assertEquals(expected, actual);
-	}
-	
-	@ParameterizedTest
-	@CsvSource({"i,1", "v,5", "x,10", "l,50", "c,100", "d,500", "m,1000"})
-	void testStringToIntValidLowercaseRomanSymbol(String romanNumeral, int expected) {
-		int actual = convertFromStringToInt(romanNumeral);
-		assertEquals(expected, actual);
-	}
-	
-	@ParameterizedTest
-	@ValueSource(strings = {"@", "V!", "123", "ten", "vi"})
-	void testStringToIntInvalidRomanNumeral(String invalidInput) {
-		assertThrows(IllegalArgumentException.class,
-				() -> convertFromStringToInt(invalidInput));
-	}
-	
-	@ParameterizedTest
-	@ValueSource(strings = {"IIII", "VIIII", "xxxx", "CMXCIX", "IM"})
-	void testStringToIntNonStandardRomanNumeral(String invalidInput) {
-		assertThrows(IllegalArgumentException.class,
-				() -> convertFromStringToInt(invalidInput));
-	}
-
-	@ValueSource(ints = {0, -1, 10000})
-	void testConvertFromIntToStringThrowsExceptionForOutOfRange(int n) {
-		assertThrows(IllegalArgumentException.class,
-				() -> convertFromIntToString(n));
-	}
 	
-	@ParameterizedTest
-	@ValueSource(strings = {"DCCCC", "IIII", "VIIII", "XXXX"})
-	void testThrowsExceptionForNonStandardForm(String s) {
-		assertThrows(IllegalArgumentException.class,
-				() -> convertFromStringToInt(s));
-	}
 	
+	//Does the test convert Strings to ints?
 	@ParameterizedTest
-	@ValueSource(ints = {-1, 0, 10000})
-	void testOutOfBoundsNumbers(int num) {
-		assertThrows(IllegalArgumentException.class,
-				() -> convertFromIntToString(num));
+	@CsvSource({"II,2.0","III,4", "VI,5", "XII,12"})
+	void string2IntConvert(String expected, int actual)
```

Comment:
The second argument is the expected value, not the first. The first could be named `s` or `text`.